src/app/storymaps/tpl/core/MainView.js

```diff
@@ -77,7 +79,27 @@ define(["lib-build/css!./MainView",
       this.init = function(core)
       {
         _core = core;
+        // After a map is loaded (when the map starts to render)
+        topic.subscribe("story-loaded-map", function(result){
+          if ( result.index !== null )
+            console.log("The map", result.id, "has been loaded from the entry", result.index);
+
+          // *******************************************
+          // **** Maptiks Changes below
+          // *******************************************
+          container = $(app.maps[result.id].response.map.container); // the current map div
```

Comment:
use proper JS form... should declare the container variable

ie: var container